en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4191 @@
+<td>
+
+NOTE: This operations can only be invoked via using an on-prem key as the authentication header.
+
+Returns a list of successful hit counts (i.e., 2xx response codes) within the specified time range for each API,
+grouped by each application. Optionally, you can configure the <code>successAPIUsageByAppFilter</code> parameter to further
+filter the results.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">timeFilter</td>
+<td valign="top"><a href="#timefilter">TimeFilter</a>!</td>
+<td>
+
+Time filter
+
+</td>
+</tr>
+<tr>
+<td colspan="2" align="right" valign="top">successAPIUsageByAppFilter</td>
+<td valign="top"><a href="#successapiusagebyappfilter">SuccessAPIUsageByAppFilter</a></td>
+<td>
+
+Successful API usage by application filter.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+## Objects
+
+### API
+
+Represents API details.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>id</strong></td>
+<td valign="top"><a href="#id">ID</a>!</td>
+<td>
+
+The ID of the API.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>name</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+API name.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>version</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+API version.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>provider</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+The API provider name.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### APILatency
+
+Represents latency values for a single API over time.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>summary</strong></td>
+<td valign="top">[<a href="#latency">Latency</a>]</td>
+<td>
+
+A list containing latency category values over time. Returns an empty array if no data is available.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>granularity</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+The granularity value for which data is retrieved.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### APIUsage
+
+Represents the API usage for a single timestamp. The timestamp is calculated based on the granularity value provided.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>timeSpan</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Represents a timestamp value that is calculated based on the specified granularity value. For example, granularity
```

Comment:
Change "For example, granularity value is..." to "For example, if the granularity value is..." Check all the places that use this phrase.